### PR TITLE
Add GPTReview With GitHub Example

### PR DESCRIPTION
**Pull request by techmaharaj**
Adding example for GPTReview using GitHub Actions

### CODE REVIEW

**Comment by techmaharaj** [comment]
@cloudnautique can you please review this?

File: examples/gptreview-ghaction/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Pre-requisites
+
+- GitHub Account
+- OpenAI API Key
+
+## How To Run This Example
+
+- Create a new repository in your GitHub account and create a `codereview.gpt` file in that repo based on the contents provided in this file.
+- Congfigure a GitHub Action for that repository and copy the contents from `workflow.yaml` to your `main.yaml`.
```
Comment (cloudnautique) [inline examples/gptreview-ghaction/README.md:18]: Does this mean the `.github/workflows/` path?
If it does, let's make sure to specify more detail for users who are not familiar with GH Actions.

File: examples/gptreview-ghaction/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- `workflow.yaml`: The workflow file for the GitHub action.
+
+## Pre-requisites
+
+- GitHub Account
+- OpenAI API Key
+
+## How To Run This Example
+
+- Create a new repository in your GitHub account and create a `codereview.gpt` file in that repo based on the contents provided in this file.
```
Comment (cloudnautique) [inline examples/gptreview-ghaction/README.md:17]: Call out that is in the root of the repo.

File: examples/gptreview-ghaction/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Congfigure a GitHub Action for that repository and copy the contents from `workflow.yaml` to your `main.yaml`.
+- Configure your `OPENAI_API_KEY` and `GH_TOKEN` as environment variables.
```
Comment (cloudnautique) [inline examples/gptreview-ghaction/README.md:19]: Can you provide links on how to set these as secrets and environment variables? I'm assuming this means in the GH repo so they are added at runtime with the Action?

File: examples/gptreview-ghaction/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- Add some code file to the repository and open a new pull request.
```
Comment (cloudnautique) [inline examples/gptreview-ghaction/README.md:20]: What happens if they push to `main` branch?

File: examples/gptreview-ghaction/README.md (hunk 1 above)
Comment (techmaharaj) [reply examples/gptreview-ghaction/README.md]: Done.

File: examples/gptreview-ghaction/README.md (hunk 3 above)
Comment (techmaharaj) [reply examples/gptreview-ghaction/README.md]: Added a link on how to setup environment variables for a repo.

File: examples/gptreview-ghaction/README.md (hunk 4 above)
Comment (techmaharaj) [reply examples/gptreview-ghaction/README.md]: It should still work, as the action triggers on new pull requests irrespective of the branch. Having said that, I've mentioned to create a new branch and push changes to it.

File: examples/gptreview-ghaction/README.md (hunk 2 above)
Comment (techmaharaj) [reply examples/gptreview-ghaction/README.md]: Done.

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**